connectors/src/connectors/github/index.ts

```diff
@@ -224,44 +213,33 @@ export async function fullResyncGithubConnector(
 export async function cleanupGithubConnector(
   connectorId: ModelId
 ): Promise<Result<undefined, Error>> {
-  return sequelizeConnection.transaction(async (transaction) => {
-    try {
-      const connector = await ConnectorModel.findOne({
-        where: {
-          id: connectorId,
-        },
-        transaction,
-      });
+  const connector = await ConnectorResource.fetchById(connectorId);
```

Comment:
So no transaction here is a bit different right ?

Reply:
Done!

Reply:
Seems like it's still here?